src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -85,6 +101,9 @@ const DefaultTranscriptslist = (props: Props) => {
           {gene.transcripts.length > 5 && !isFilterOpen && (
             <div className={styles.filterLabel} onClick={toggleFilter}>
               Filter & sort
+              {shouldShowIndicator() && (
+                <span className={styles.activeIndicator}></span>
```

Comment:
self-closing span?

Reply:
I am curious as to why self-closing span is better in this case as HTML/CSS purists wouldn't like it 😉  And also wondering if we need to make Prettier do it for us automatically.

Reply:
A self-closing tag just communicates more explicitly that this tag isn't intended to contain anything. Which arguably makes it preferable for React purists 😄 Definitely a style thing. I probably shouldn't have left the comment to that line. I guess a more interesting question is whether to represent the indicator as a stand-alone empty span or as a styled pseudoelement.

Reply:
✅

Reply:
You'd need `content: ''` for the `after` to work 🙂 

But I guess I was wrong, and using a span really is better here. Either that, or wrap the label text into a span of its own. Because otherwise we are getting this:

![image](https://user-images.githubusercontent.com/6834224/102806101-2d699300-43b4-11eb-9ebc-553d54053105.png)

Sorry for wasting your time 🙁

Reply:
Ah that's fine. It didn't take long to switch it back to using `span` 😄

Reply:
✅